docs/api/auto-updater.md

```diff
@@ -50,6 +50,11 @@ The server-side setup is also different from OS X. You can read the documents of
 There is no built-in support for auto-updater on Linux, so it is recommended to
 use the distribution's package manager to update your app.
 
+## Usage
+```javascript
+var autoUpdater = require('electron').remote.autoUpdater;
```

Comment:
As soon as you mentioned that it became blindingly obvious what was going on, and I feel extremely stupid for taking that long to figure it out. I'm closing this issue, since now that I realize what I did wrong I don't actually think it's helpful.